src/test/resources/dp/xlsx/v4_2_generational_income.csv

```diff
@@ -0,0 +1,37 @@
+V4_1,Data Marking,yyyy-yy,Time,uk-only,Geography,single-year-of-age,Age,tax-benefit-type,TypeOfTaxOrBenefit,decade,Decade
```

Comment:
Just a quick check, I'm guessing all this information is already in the public domain so there's nothing sensitive here?

Reply:
Yes - this data was taken from version 1 which is published and publicly available.

Good question though.